package.json

```diff
@@ -39,9 +39,9 @@
 	"dependencies": {
 		"js-yaml": "^3.14.0",
 		"vscode-json-languageservice": "3.4.9",
-		"vscode-languageserver": "^6.1.1",
+		"vscode-languageserver": "^5.2.1",
```

Comment:
What's the reason for the downgrade? The VSC Toolkit is on `^6.1.4`

Reply:
The only reason was because of errors in the vscode-language-servers type definitions. When we had yaml language server `0.15.0` and vscode-langauge-server `^6.1.1` theres a spam of:

```
node_modules/vscode-languageserver/lib/main.d.ts:260:19 - error TS2315: Type 'NotificationType0' is not generic.

260     add<RO>(type: NotificationType0<RO>, registerParams: RO): void;
                      ~~~~~~~~~~~~~~~~~~~~~

node_modules/vscode-languageserver/lib/main.d.ts:261:22 - error TS2314: Generic type 'NotificationType<P>' requires 1 type argument(s).

261     add<P, RO>(type: NotificationType<P, RO>, registerParams: RO): void;

```

notifications. I just added skipLibCheck in the tsconfig so we skip type checking in the declaration files which makes more sense then downgrading the package

Reply:
`yaml-language-server` should probably be upgraded to 1.x.x but that may have breaking changes. `skipLibCheck` can be dangerous although this specific scenario is safe.

Reply:
Agreed. I think it's going to be pretty big pain to migration to the latest yaml language server version. IIRC we've done like 20 or 30 yaml-language-server releases since 0.15.0 was released, changed the underlying parser entirely and a few other things but I just wanted to keep this change minimal to make my life easier 😆